lib/errorHandlers/suppressError.ts

```diff
@@ -20,23 +20,17 @@ function createPlatformVersionError(
 ): void {
   let translationKey = 'unspecifiedPlatformVersion';
   let platformVersion = 'unspecified platformVersion';
-  const errorContext = err.data.context;
+  const errorContext = err?.data?.context;
 
-  switch (subCategory) {
-    case PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED:
-      translationKey = 'platformVersionRetired';
-      if (errorContext && errorContext[subCategory]) {
-        platformVersion = errorContext[subCategory];
-      }
-      break;
-    case PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST:
-      translationKey = 'nonExistentPlatformVersion';
-      if (errorContext && errorContext[subCategory]) {
-        platformVersion = errorContext[subCategory];
-      }
-      break;
-    default:
-      break;
+  if (subCategory === PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED) {
+    platformVersion = errorContext?.RETIRED_PLATFORM_VERSION ?? platformVersion;
+    translationKey = 'platformVersionRetired';
+  } else if (
+    subCategory ===
+    PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_SPECIFIED_DOES_NOT_EXIST
+  ) {
+    platformVersion = errorContext?.PLATFORM_VERSION ?? platformVersion;
+    translationKey = 'nonExistentPlatformVersion';
```

Comment:
When I fixed the above, I also noticed that these were incorrect.  We were looking for the existence of the wrong keys so it always just said `unspecified platformVersion` instead of listing the actual invalid platform version